src/tempo.cpp

```diff
@@ -143,22 +143,33 @@ int Tempo::InitMaxMeasureDuration(FunctorParams *functorParams)
         params->m_currentTempo = this->GetMidiBpm();
     }
     else if (this->HasMm()) {
-        double mm = this->GetMm();
-        double mmUnit = 4;
-        if (this->HasMmUnit() && (this->GetMmUnit() > DURATION_breve)) {
-            mmUnit = pow(2, (int)this->GetMmUnit() - 2);
-        }
-        if (this->HasMmDots()) {
-            double dotsUnit = 0.0;
-            for (int d = 0; d < this->GetMmDots(); d++) {
-                dotsUnit += mmUnit / 4.0 / pow(2, d);
-            }
-            mmUnit -= dotsUnit;
-        }
-        if (mmUnit > 0) params->m_currentTempo = mm * 4.0 / mmUnit;
+        double tempo = Tempo::CalcTempo(this);
+        if (tempo > 0) params->m_currentTempo = tempo;
     }
 
     return FUNCTOR_CONTINUE;
 }
 
+double Tempo::CalcTempo(const AttMmTempo *attMmTempo)
+{
+    double tempo = 0.0;
+
+    double mm = attMmTempo->GetMm();
+    double mmUnit = 4;
+
+    if (attMmTempo->HasMmUnit() && (attMmTempo->GetMmUnit() > DURATION_breve)) {
+        mmUnit = pow(2, (int)attMmTempo->GetMmUnit() - 2);
+    }
+    if (attMmTempo->HasMmDots()) {
+        double dotsUnit = 0.0;
+        for (int d = 0; d < attMmTempo->GetMmDots(); d++) {
+            dotsUnit += mmUnit / 4.0 / pow(2, d);
+        }
+        mmUnit -= dotsUnit;
+    }
+    if (mmUnit > 0) tempo = mm * 4.0 / mmUnit;
+
+    return tempo;
```

Comment:
Instead of returning a tempo of zero this should return Verovio's default tempo. Then you can drop the extra-check if it's not 0.

Reply:
Sure, thanks! I have also amended the attMmTempo call at `ExportMidi()`.

Reply:
One more thing: `scoreDef@midi.bpm` and `scoreDef@mm is` only read at the beginning of an encoding (both for timemap and MIDI output). It would be good to accept tempo changes from scoreDef also during an encoding. 

This commit c222ac8 addresses this. As I do not oversea all the MIDI export steps, could you please check whether this is general enough? 

<details><summary>It works for this example:</summary>

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
   <meiHead>
      <fileDesc>
         <titleStmt>
            <title>Eine Kleine Nachtmusik</title>
            <respStmt/>
         </titleStmt>
         <pubStmt/>
      </fileDesc>
      <encodingDesc xml:id="encodingdesc-0000001754878255">
         <appInfo xml:id="appinfo-0000000698423887">
            <application xml:id="application-0000000264654307" isodate="2020-02-28T11:52:02" version="2.5.0-c6ee579">
               <name xml:id="name-0000000606304812">Verovio</name>
               <p xml:id="p-0000000355070269">Transcoded from MusicXML</p>
            </application>
            <application xml:id="aza35hs1" startdate="2023-01-31T13:06:11" version="0.8.2" enddate="2023-01-31T13:08:29">
               <name xml:id="nb25h731">mei-friend</name>
               <p xml:id="pqn2bxi2">First edit by mei-friend 0.8.2, 27 Jan 2023.</p>
            </application>
         </appInfo>
      </encodingDesc>
   </meiHead>
    <music>
        <body>
            <mdiv xml:id="mdiv-0000001956439717">
                <score xml:id="score-0000001760204402">
                    <scoreDef xml:id="scoredef-0000000556440418" mm="184" mm.unit="4">
                        <staffGrp xml:id="staffgrp-0000000649239688">
                            <staffGrp xml:id="staffgrp-0000000407025809" symbol="bracket">
                                <staffDef xml:id="staffdef-0000001753169665" n="1" scale="112.00%" lines="5" ppq="24" clef.shape="G" clef.line="2" meter.count="4" meter.unit="4">
                                    <keySig xml:id="keysig-0000000737640739" mode="major" sig="1s" />
                                    <label xml:id="label-0000001147356168">Violine I</label>
                                    <instrDef xml:id="instrdef-0000001183562108" midi.channel="0" midi.instrnum="48" midi.volume="80.00%" />
                                </staffDef>
                                <staffDef xml:id="staffdef-0000000472472902" n="2" scale="112.00%" lines="5" ppq="24" clef.shape="G" clef.line="2" meter.count="4" meter.unit="4">
                                    <keySig xml:id="keysig-0000000395118890" mode="major" sig="1s" />
                                    <label xml:id="label-0000001406630613">Violine II</label>
                                    <instrDef xml:id="instrdef-0000001220814524" midi.channel="1" midi.instrnum="48" midi.volume="80.00%" />
                                </staffDef>
                                <staffDef xml:id="staffdef-0000001263472404" n="3" scale="112.00%" lines="5" ppq="4" clef.shape="C" clef.line="3" meter.count="4" meter.unit="4">
                                    <keySig xml:id="keysig-0000001133519705" mode="major" sig="1s" />
                                    <label xml:id="label-0000001890037757">Viola</label>
                                    <instrDef xml:id="instrdef-0000000134971751" midi.channel="2" midi.instrnum="48" midi.volume="80.00%" />
                                </staffDef>
                                <staffDef xml:id="staffdef-0000001990347289" n="4" scale="112.00%" lines="5" ppq="4" clef.shape="F" clef.line="4" meter.count="4" meter.unit="4">
                                    <keySig xml:id="keysig-0000001953695774" mode="major" sig="1s" />
                                    <label xml:id="label-0000001402864151">Cello/Bass</label>
                                    <instrDef xml:id="instrdef-0000001047715990" midi.channel="3" midi.instrnum="48" midi.volume="80.00%" />
                                </staffDef>
                            </staffGrp>
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-0000000020663342">
                        <pb xml:id="pb-0000001543921827" />
                        <measure xml:id="measure-0000000541918754" n="1">
                            <staff xml:id="staff-0000001820325675" n="1">
                                <layer xml:id="layer-0000001396147484" n="1">
                                    <chord xml:id="chord-0000002076630819" dur.ppq="24" dur="4" cue="false" stem.dir="down">
                                        <note xml:id="note-0000001644436466" oct="4" pname="d" />
                                        <note xml:id="note-0000001029943889" oct="4" pname="b" />
                                        <note xml:id="note-0000001548747603" oct="5" pname="g" />
                                    </chord>
                                    <rest xml:id="rest-0000000151678334" dur.ppq="12" dur="8" />
                                    <note xml:id="note-0000000194670549" dur.ppq="12" dur="8" oct="5" pname="d" stem.dir="down" />
                                    <note xml:id="note-0000001210322662" dur.ppq="24" dur="4" oct="5" pname="g" stem.dir="down" />
                                    <rest xml:id="rest-0000000927875850" dur.ppq="12" dur="8" />
                                    <note xml:id="note-0000001930650083" dur.ppq="12" dur="8" oct="5" pname="d" stem.dir="down" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000000366118595" n="2">
                                <layer xml:id="layer-0000002011264209" n="1">
                                    <chord xml:id="chord-0000000909190526" dur.ppq="24" dur="4" cue="false" stem.dir="down">
                                        <note xml:id="note-0000001924956883" oct="4" pname="d" />
                                        <note xml:id="note-0000001419022077" oct="4" pname="b" />
                                        <note xml:id="note-0000001698148204" oct="5" pname="g" />
                                    </chord>
                                    <rest xml:id="rest-0000000719195998" dur.ppq="12" dur="8" />
                                    <note xml:id="note-0000001489173070" dur.ppq="12" dur="8" oct="5" pname="d" stem.dir="down" />
                                    <note xml:id="note-0000001757365352" dur.ppq="24" dur="4" oct="5" pname="g" stem.dir="down" />
                                    <rest xml:id="rest-0000001696873873" dur.ppq="12" dur="8" />
                                    <note xml:id="note-0000000776351351" dur.ppq="12" dur="8" oct="5" pname="d" stem.dir="down" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000001595994033" n="3">
                                <layer xml:id="layer-0000001130253914" n="1">
                                    <note xml:id="note-0000001684674883" dur.ppq="4" dur="4" oct="4" pname="g" stem.dir="down" />
                                    <rest xml:id="rest-0000001906356533" dur.ppq="2" dur="8" />
                                    <note xml:id="note-0000001825720538" dur.ppq="2" dur="8" oct="4" pname="d" stem.dir="down" />
                                    <note xml:id="note-0000001638733830" dur.ppq="4" dur="4" oct="4" pname="g" stem.dir="down" />
                                    <rest xml:id="rest-0000000721708035" dur.ppq="2" dur="8" />
                                    <note xml:id="note-0000000759305989" dur.ppq="2" dur="8" oct="4" pname="d" stem.dir="down" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000000240859524" n="4">
                                <layer xml:id="layer-0000000506295766" n="1">
                                    <note xml:id="note-0000000982729748" dur.ppq="4" dur="4" oct="3" pname="g" stem.dir="down" />
                                    <rest xml:id="rest-0000000442145559" dur.ppq="2" dur="8" />
                                    <note xml:id="note-0000000846991493" dur.ppq="2" dur="8" oct="3" pname="d" stem.dir="down" />
                                    <note xml:id="note-0000001864410535" dur.ppq="4" dur="4" oct="3" pname="g" stem.dir="down" />
                                    <rest xml:id="rest-0000001213968368" dur.ppq="2" dur="8" />
                                    <note xml:id="note-0000002071714476" dur.ppq="2" dur="8" oct="3" pname="d" stem.dir="down" />
                                </layer>
                            </staff>
                            <dir xml:id="dir-0000001161584563" place="above" staff="1" tstamp="1.000000" vgrp="225">
                                <rend xml:id="rend-0000000339874379" fontweight="bold">Allegro (scoreDef mm=184)</rend>
                            </dir>
                            <dynam xml:id="dynam-0000002109670480" place="below" staff="1" tstamp="1.000000" vgrp="86">f</dynam>
                            <dynam xml:id="dynam-0000000814577510" place="below" staff="2" tstamp="1.000000" vgrp="87">f</dynam>
                            <dynam xml:id="dynam-0000001800961601" place="below" staff="3" tstamp="1.000000" vgrp="76">f</dynam>
                            <dynam xml:id="dynam-0000000119345273" place="below" staff="4" tstamp="1.000000" vgrp="77">f</dynam>
                        </measure>
                        <measure xml:id="measure-0000002105522458" n="2">
                            <staff xml:id="staff-0000000274167548" n="1">
                                <layer xml:id="layer-0000001581556421" n="1">
                                    <beam xml:id="beam-0000001813668828">
                                        <note xml:id="note-0000000949106678" dur.ppq="12" dur="8" oct="5" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000000127407230" dur.ppq="12" dur="8" oct="5" pname="d" stem.dir="down" />
                                        <note xml:id="note-0000000292118551" dur.ppq="12" dur="8" oct="5" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000000488869615" dur.ppq="12" dur="8" oct="5" pname="b" stem.dir="down" />
                                    </beam>
                                    <note xml:id="note-0000000159185883" dur.ppq="24" dur="4" oct="6" pname="d" stem.dir="down" />
                                    <rest xml:id="rest-0000001819995066" dur.ppq="24" dur="4" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000001146249335" n="2">
                                <layer xml:id="layer-0000002084259755" n="1">
                                    <beam xml:id="beam-0000000400452421">
                                        <note xml:id="note-0000000190090049" dur.ppq="12" dur="8" oct="5" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000001535270454" dur.ppq="12" dur="8" oct="5" pname="d" stem.dir="down" />
                                        <note xml:id="note-0000001274501673" dur.ppq="12" dur="8" oct="5" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000001547722933" dur.ppq="12" dur="8" oct="5" pname="b" stem.dir="down" />
                                    </beam>
                                    <note xml:id="note-0000000109918820" dur.ppq="24" dur="4" oct="6" pname="d" stem.dir="down" />
                                    <rest xml:id="rest-0000000569671320" dur.ppq="24" dur="4" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000001639870490" n="3">
                                <layer xml:id="layer-0000000498199832" n="1">
                                    <beam xml:id="beam-0000000205836771">
                                        <note xml:id="note-0000002049938527" dur.ppq="2" dur="8" oct="4" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000001236674468" dur.ppq="2" dur="8" oct="4" pname="d" stem.dir="down" />
                                        <note xml:id="note-0000001441048010" dur.ppq="2" dur="8" oct="4" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000000373333204" dur.ppq="2" dur="8" oct="4" pname="b" stem.dir="down" />
                                    </beam>
                                    <note xml:id="note-0000001811426741" dur.ppq="4" dur="4" oct="5" pname="d" stem.dir="down" />
                                    <rest xml:id="rest-0000001921056115" dur.ppq="4" dur="4" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000000289854943" n="4">
                                <layer xml:id="layer-0000001099115605" n="1">
                                    <beam xml:id="beam-0000000181641741">
                                        <note xml:id="note-0000001278478600" dur.ppq="2" dur="8" oct="3" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000001815941965" dur.ppq="2" dur="8" oct="3" pname="d" stem.dir="down" />
                                        <note xml:id="note-0000000499014591" dur.ppq="2" dur="8" oct="3" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000001014589402" dur.ppq="2" dur="8" oct="3" pname="b" stem.dir="down" />
                                    </beam>
                                    <note xml:id="note-0000001183922234" dur.ppq="4" dur="4" oct="4" pname="d" stem.dir="down" />
                                    <rest xml:id="rest-0000001744997383" dur.ppq="4" dur="4" />
                                </layer>
                            </staff>
                           <tempo xml:id="tuyum3i1" startid="#note-0000000949106678" place="above" mm="99">mm=99</tempo>
                       </measure>
                       <scoreDef mm="66"></scoreDef>
                        <measure xml:id="measure-0000002047490041" n="3">
                            <staff xml:id="staff-0000001798641468" n="1">
                                <layer xml:id="layer-0000001787337504" n="1">
                                    <note xml:id="note-0000000780175492" dur.ppq="24" dur="4" oct="6" pname="c" stem.dir="down" />
                                    <rest xml:id="rest-0000002021829109" dur.ppq="12" dur="8" />
                                    <note xml:id="note-0000001248088482" dur.ppq="12" dur="8" oct="5" pname="a" stem.dir="down" />
                                    <note xml:id="note-0000000002853078" dur.ppq="24" dur="4" oct="6" pname="c" stem.dir="down" />
                                    <rest xml:id="rest-0000000707041712" dur.ppq="12" dur="8" />
                                    <note xml:id="note-0000001223034733" dur.ppq="12" dur="8" oct="5" pname="a" stem.dir="down" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000001772176335" n="2">
                                <layer xml:id="layer-0000001516962102" n="1">
                                    <note xml:id="note-0000000656191130" dur.ppq="24" dur="4" oct="6" pname="c" stem.dir="down" />
                                    <rest xml:id="rest-0000001275794565" dur.ppq="12" dur="8" />
                                    <note xml:id="note-0000001802522307" dur.ppq="12" dur="8" oct="5" pname="a" stem.dir="down" />
                                    <note xml:id="note-0000000440605520" dur.ppq="24" dur="4" oct="6" pname="c" stem.dir="down" />
                                    <rest xml:id="rest-0000000733359784" dur.ppq="12" dur="8" />
                                    <note xml:id="note-0000001169239555" dur.ppq="12" dur="8" oct="5" pname="a" stem.dir="down" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000000846507094" n="3">
                                <layer xml:id="layer-0000000165567483" n="1">
                                    <note xml:id="note-0000001701363916" dur.ppq="4" dur="4" oct="5" pname="c" stem.dir="down" />
                                    <rest xml:id="rest-0000001078576407" dur.ppq="2" dur="8" />
                                    <note xml:id="note-0000000724208122" dur.ppq="2" dur="8" oct="4" pname="a" stem.dir="down" />
                                    <note xml:id="note-0000001976078905" dur.ppq="4" dur="4" oct="5" pname="c" stem.dir="down" />
                                    <rest xml:id="rest-0000001123555480" dur.ppq="2" dur="8" />
                                    <note xml:id="note-0000000773244289" dur.ppq="2" dur="8" oct="4" pname="a" stem.dir="down" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000000698867468" n="4">
                                <layer xml:id="layer-0000001277469233" n="1">
                                    <note xml:id="note-0000002031379972" dur.ppq="4" dur="4" oct="4" pname="c" stem.dir="down" />
                                    <rest xml:id="rest-0000000708169398" dur.ppq="2" dur="8" />
                                    <note xml:id="note-0000000848700512" dur.ppq="2" dur="8" oct="3" pname="a" stem.dir="down" />
                                    <note xml:id="note-0000000523121810" dur.ppq="4" dur="4" oct="4" pname="c" stem.dir="down" />
                                    <rest xml:id="rest-0000000310209852" dur.ppq="2" dur="8" />
                                    <note xml:id="note-0000001754171295" dur.ppq="2" dur="8" oct="3" pname="a" stem.dir="down" />
                                </layer>
                            </staff>
                           <dir xml:id="dhi55zk2" startid="#note-0000000780175492" place="above">scoreDef mm="66"</dir>
                       </measure>
                        <measure xml:id="measure-0000001978772094" n="4">
                            <staff xml:id="staff-0000000910504925" n="1">
                                <layer xml:id="layer-0000002035289600" n="1">
                                    <beam xml:id="beam-0000001992777784">
                                        <note xml:id="note-0000000461257076" dur.ppq="12" dur="8" oct="6" pname="c" stem.dir="down" />
                                        <note xml:id="note-0000002079194309" dur.ppq="12" dur="8" oct="5" pname="a" stem.dir="down" />
                                        <note xml:id="note-0000001164847379" dur.ppq="12" dur="8" oct="5" pname="f" stem.dir="down" accid.ges="s" />
                                        <note xml:id="note-0000001627845162" dur.ppq="12" dur="8" oct="5" pname="a" stem.dir="down" />
                                    </beam>
                                    <note xml:id="note-0000000251974954" dur.ppq="24" dur="4" oct="5" pname="d" stem.dir="down" />
                                    <rest xml:id="rest-0000000105299994" dur.ppq="24" dur="4" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000001527965960" n="2">
                                <layer xml:id="layer-0000000914438894" n="1">
                                    <beam xml:id="beam-0000001581513526">
                                        <note xml:id="note-0000001092732563" dur.ppq="12" dur="8" oct="6" pname="c" stem.dir="down" />
                                        <note xml:id="note-0000000276037197" dur.ppq="12" dur="8" oct="5" pname="a" stem.dir="down" />
                                        <note xml:id="note-0000000792492459" dur.ppq="12" dur="8" oct="5" pname="f" stem.dir="down" accid.ges="s" />
                                        <note xml:id="note-0000000380102156" dur.ppq="12" dur="8" oct="5" pname="a" stem.dir="down" />
                                    </beam>
                                    <note xml:id="note-0000001760569714" dur.ppq="24" dur="4" oct="5" pname="d" stem.dir="down" />
                                    <rest xml:id="rest-0000001865494832" dur.ppq="24" dur="4" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000001621599780" n="3">
                                <layer xml:id="layer-0000000512538383" n="1">
                                    <beam xml:id="beam-0000000675694964">
                                        <note xml:id="note-0000000511734612" dur.ppq="2" dur="8" oct="5" pname="c" stem.dir="down" />
                                        <note xml:id="note-0000000051617649" dur.ppq="2" dur="8" oct="4" pname="a" stem.dir="down" />
                                        <note xml:id="note-0000002101917002" dur.ppq="2" dur="8" oct="4" pname="f" stem.dir="down" accid.ges="s" />
                                        <note xml:id="note-0000000651965587" dur.ppq="2" dur="8" oct="4" pname="a" stem.dir="down" />
                                    </beam>
                                    <note xml:id="note-0000001124053715" dur.ppq="4" dur="4" oct="4" pname="d" stem.dir="down" />
                                    <rest xml:id="rest-0000000557145346" dur.ppq="4" dur="4" />
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000001502855946" n="4">
                                <layer xml:id="layer-0000001944712055" n="1">
                                    <beam xml:id="beam-0000000074401045">
                                        <note xml:id="note-0000000622880761" dur.ppq="2" dur="8" oct="4" pname="c" stem.dir="down" />
                                        <note xml:id="note-0000001921654649" dur.ppq="2" dur="8" oct="3" pname="a" stem.dir="down" />
                                        <note xml:id="note-0000001243118510" dur.ppq="2" dur="8" oct="3" pname="f" stem.dir="down" accid.ges="s" />
                                        <note xml:id="note-0000000440724817" dur.ppq="2" dur="8" oct="3" pname="a" stem.dir="down" />
                                    </beam>
                                    <note xml:id="note-0000000590900816" dur.ppq="4" dur="4" oct="3" pname="d" stem.dir="down" />
                                    <rest xml:id="rest-0000001305630784" dur.ppq="4" dur="4" />
                                </layer>
                            </staff>
                           <tempo xml:id="thh5r54" startid="#note-0000000461257076" place="above" mm="77">mm=77</tempo>
                       </measure>
                        <measure xml:id="measure-0000000250474030" n="5">
                            <staff xml:id="staff-0000000718143814" n="1">
                                <layer xml:id="layer-0000000984985758" n="1">
                                    <chord xml:id="chord-0000002061240733" dur.ppq="12" dur="8" cue="false" stem.dir="down">
                                        <note xml:id="note-0000001851683630" oct="4" pname="d" />
                                        <note xml:id="note-0000000066806127" oct="4" pname="b" />
                                        <note xml:id="note-0000001824112755" oct="5" pname="g" />
                                    </chord>
                                    <rest xml:id="rest-0000000386528713" dur.ppq="12" dur="8" />
                                    <note xml:id="note-0000000250047216" dur.ppq="24" dur="4" oct="5" pname="g" stem.dir="down" />
                                    <beam xml:id="beam-0000001554770705">
                                        <note xml:id="note-0000000450222239" dur.ppq="12" dur="8" oct="5" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000001300282492" dur.ppq="12" dur="8" oct="5" pname="b" stem.dir="down" />
                                        <note xml:id="note-0000002059360054" dur.ppq="12" dur="8" oct="5" pname="a" stem.dir="down" />
                                        <note xml:id="note-0000000670488879" dur.ppq="12" dur="8" oct="5" pname="g" stem.dir="down" />
                                    </beam>
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000001899938286" n="2">
                                <layer xml:id="layer-0000001328425559" n="1">
                                    <beam xml:id="beam-0000001608375901">
                                        <chord xml:id="chord-0000001117361773" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000001597103318" oct="3" pname="b" />
                                            <note xml:id="note-0000001902309443" oct="4" pname="d" />
                                        </chord>
                                        <chord xml:id="chord-0000001065320635" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000000378271965" oct="3" pname="b" />
                                            <note xml:id="note-0000001272747406" oct="4" pname="d" />
                                        </chord>
                                        <chord xml:id="chord-0000001396277528" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000002128528522" oct="3" pname="b" />
                                            <note xml:id="note-0000001682602327" oct="4" pname="d" />
                                        </chord>
                                        <chord xml:id="chord-0000000897915587" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000001432646193" oct="3" pname="b" />
                                            <note xml:id="note-0000000899683240" oct="4" pname="d" />
                                        </chord>
                                    </beam>
                                    <beam xml:id="beam-0000000543856153">
                                        <chord xml:id="chord-0000000931302252" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000000899961839" oct="3" pname="b" />
                                            <note xml:id="note-0000001536130028" oct="4" pname="d" />
                                        </chord>
                                        <chord xml:id="chord-0000000393891510" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000000688976362" oct="3" pname="b" />
                                            <note xml:id="note-0000001590008516" oct="4" pname="d" />
                                        </chord>
                                        <chord xml:id="chord-0000000694578143" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000002134108791" oct="3" pname="b" />
                                            <note xml:id="note-0000000053744309" oct="4" pname="d" />
                                        </chord>
                                        <chord xml:id="chord-0000001140620612" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000001337469623" oct="3" pname="b" />
                                            <note xml:id="note-0000001971592762" oct="4" pname="d" />
                                        </chord>
                                    </beam>
                                    <beam xml:id="beam-0000000886877724">
                                        <chord xml:id="chord-0000000361647978" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000000069913441" oct="3" pname="b" />
                                            <note xml:id="note-0000000838845236" oct="4" pname="d" />
                                        </chord>
                                        <chord xml:id="chord-0000002014065402" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000000241738897" oct="3" pname="b" />
                                            <note xml:id="note-0000001759967400" oct="4" pname="d" />
                                        </chord>
                                        <chord xml:id="chord-0000000000169907" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000000332338022" oct="3" pname="b" />
                                            <note xml:id="note-0000000708143302" oct="4" pname="d" />
                                        </chord>
                                        <chord xml:id="chord-0000001360384057" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000000410105040" oct="3" pname="b" />
                                            <note xml:id="note-0000001863940037" oct="4" pname="d" />
                                        </chord>
                                    </beam>
                                    <beam xml:id="beam-0000001896243070">
                                        <chord xml:id="chord-0000000450127937" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000001499956010" oct="3" pname="b" />
                                            <note xml:id="note-0000001862832425" oct="4" pname="d" />
                                        </chord>
                                        <chord xml:id="chord-0000001859442993" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000000460477362" oct="3" pname="b" />
                                            <note xml:id="note-0000001476352207" oct="4" pname="d" />
                                        </chord>
                                        <chord xml:id="chord-0000001780396902" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000001025485611" oct="3" pname="b" />
                                            <note xml:id="note-0000000093594616" oct="4" pname="d" />
                                        </chord>
                                        <chord xml:id="chord-0000001655170868" dur.ppq="6" dur="16" cue="false" stem.dir="up">
                                            <note xml:id="note-0000001086681508" oct="3" pname="b" />
                                            <note xml:id="note-0000002101098885" oct="4" pname="d" />
                                        </chord>
                                    </beam>
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000001959996302" n="3">
                                <layer xml:id="layer-0000001406186381" n="1">
                                    <beam xml:id="beam-0000000716970232">
                                        <note xml:id="note-0000000587945907" dur.ppq="2" dur="8" oct="4" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000001034599102" dur.ppq="2" dur="8" oct="4" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000000332017555" dur.ppq="2" dur="8" oct="4" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000001056531979" dur.ppq="2" dur="8" oct="4" pname="g" stem.dir="down" />
                                    </beam>
                                    <beam xml:id="beam-0000001738177657">
                                        <note xml:id="note-0000001331831058" dur.ppq="2" dur="8" oct="4" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000000862539125" dur.ppq="2" dur="8" oct="4" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000001180456625" dur.ppq="2" dur="8" oct="4" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000001480565389" dur.ppq="2" dur="8" oct="4" pname="g" stem.dir="down" />
                                    </beam>
                                </layer>
                            </staff>
                            <staff xml:id="staff-0000001130549610" n="4">
                                <layer xml:id="layer-0000000211986614" n="1">
                                    <beam xml:id="beam-0000000183651125">
                                        <note xml:id="note-0000000690457136" dur.ppq="2" dur="8" oct="3" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000001658940011" dur.ppq="2" dur="8" oct="3" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000001024575876" dur.ppq="2" dur="8" oct="3" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000001522866286" dur.ppq="2" dur="8" oct="3" pname="g" stem.dir="down" />
                                    </beam>
                                    <beam xml:id="beam-0000001103563856">
                                        <note xml:id="note-0000001928952300" dur.ppq="2" dur="8" oct="3" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000001488170988" dur.ppq="2" dur="8" oct="3" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000002095242354" dur.ppq="2" dur="8" oct="3" pname="g" stem.dir="down" />
                                        <note xml:id="note-0000000301400172" dur.ppq="2" dur="8" oct="3" pname="g" stem.dir="down" />
                                    </beam>
                                </layer>
                            </staff>
                            <tie xml:id="tie-0000002065545780" startid="#note-0000000250047216" endid="#note-0000000450222239" />
                            <slur xml:id="slur-0000001054251172" startid="#note-0000001300282492" endid="#note-0000002059360054" curvedir="above" />
                        	<tie xml:id="tie-0000000213869140" startid="#note-0000000670488879" endid="#note-0000001357685813" />
                        	<slur xml:id="slur-0000000087967475" startid="#note-0000001902309443" endid="#note-0000000309077422" curvedir="above"/>
                        	<slur xml:id="slur-0000000872367018" startid="#note-0000001597103318" endid="#note-0000001330195829" curvedir="below" />
                        	<slur xml:id="slur-0000000957298694" startid="#note-0000001480565389" endid="#note-0000001959308478" />
                        	<slur xml:id="slur-0000000117256322" startid="#note-0000000587945907" endid="#note-0000001715088183" curvedir="below" />
                        	<tie xml:id="tie-0000000851916456" startid="#note-0000000301400172" endid="#note-0000000524644218" />
                        	<slur xml:id="slur-0000000340286934" startid="#note-0000000690457136" endid="#note-0000001093435548" curvedir="below" />
                        </measure>
                     </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```
</details>
<img width="1707" alt="Bildschirmfoto 2023-01-31 um 13 58 16" src="https://user-images.githubusercontent.com/45632600/215766748-14ae990f-f960-4311-884d-cef5e4710e07.png">